python/tests/plugins/validators/test__validate_usecase_participants.py

```diff
@@ -13,8 +13,8 @@
 
 class TestUsecaseParticipants(ActiveContextTestCase):
     def test_validate_usecase_with_referenced_participants(self):
-        test_participant_1 = create_field_entry("user1", "SomeModel")
-        test_participant_2 = create_field_entry("user2", "SomeOtherModel")
+        test_participant_1 = create_field_entry("user1", description="SomeModel")
```

Comment:
I'm not sure that `SomeModel` is meant to be the description for the participant field. If it is, then also need to make `SomeModel` on line 32 the description. 

But I think that is meant to be the `type` since it was the second entry.

Definitely times like this I wish the standard would be to give the argument names if you aren't going to use them all so that it is understood for sure which are being left out. I'll bring that up for discussion with the guys again.

Reply:
You may be right.  After going through all this, I'm personally not a fan of the whole `tests.helpers` construct.  IMHO...It seems like we built an API to "automate" test data creation, but it feels like we go through a lot to do something that may be simpler and more readable by just writing YAML string constants.

Please recommend the code change and I'll add it to the commit batch.